src/System.Net.NetworkInformation/tests/FunctionalTests/System.Net.NetworkInformation.Functional.Tests.nuget.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Just as a heads up, we will not want to check this file in. Maybe we can add it to the .gitignore if it keeps popping up.